File: BR-API/BR-API/Entities/Propiedad.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace BR_API.Entities
{
    public class Propiedad
    {
        public int Id { get; set; }
        public string Titulo { get; set; }
        public decimal Precio { get; set; }
        public string Imagen { get; set; }
        public string Descripcion { get; set; }
        public int Habitaciones { get; set; }
        public int WC { get; set; }
        public int Estacionamiento { get; set; }
        public DateTime Creado { get; set; }
        public int VendedorId { get; set; }
        public Vendedor Vendedor { get; set; }
    }
}

File: BR-UI/src/app/propiedades/propiedad.ts
export interface Propiedad {
    id: number
    titulo: string
    precio: number
    imagen: string
    descripcion: string
    habitaciones: number
    wc: number
    estacionamiento: number
    // creado: Date
    vendedorId: number
    vendedor: string
}

export interface PropiedadDTO {
    id: number
    titulo: string
    precio: number
    imagen: string
    descripcion: string
    habitaciones: number
    wc: number
    estacionamiento: number
    vendedorId: number
    vendedorNombre: string
    vendedorApellido: string
}

export interface PropiedadCreacionDTO {
    titulo: string
    precio: number
    imagen: File
    descripcion: string
    habitaciones: number
    wc: number
    estacionamiento: number
    vendedorId: number
}